jazz_create-serverless-service/package.json

```diff
@@ -6,7 +6,8 @@
   "private": true,
   "author": [],
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "tst": "nyc --reporter=html --reporter=text mocha"
```

Comment:
did you mean to check this in - if yes then I would put a better command like test-report or test-coverage or something

Reply:
done